core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -717,6 +717,8 @@ class WorkspaceServiceSpec
   }
 
   it should "invite a user to a workspace" in withTestDataServicesCustomSam { services =>
+    populateWorkspacePolicies(services)
```

Comment:
We didn't error before even though the user didn't have access to the workspace.